releasenotes/notes/mcp-removal.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+- |
+  **Removed** support for reading Istio configuration over the Mesh Configuration Protocol (MCP).
```

Comment:
Should we mention the config-over-XDS ( like to design doc ) ? It is experimental, but it's good to indicate a replacement may be possible if anyone depends on the removed feature.